src/modclub_assets/src/utils/gtm.ts

```diff
@@ -0,0 +1,18 @@
+type GTMEvent = {
+  event: string;
+  [key: string]: any;
+};
+
+const GTMManager = {
+  _pushToDataLayer: (event: GTMEvent): void => {
+    if (window && window.dataLayer) {
+      window.dataLayer.push(event);
+    }
+  },
+
+  trackEvent: (eventName: string, eventData: object): void => {
```

Comment:
- Will we track events for local network? 
- Do we need a tag for env ? ( prod/dev/qa ) ?

Reply:
you need do this through the google analytics interface